src/main/java/gmailgdogra/Officer.java

```diff
@@ -6,8 +6,8 @@
     private final String lastName;
 
     public Officer(String firstName, String lastName) {
-        this.firstName = firstName;
-        this.lastName = lastName;
+        this.firstName = firstName.substring(0, 1).toUpperCase() + firstName.substring(1);
+        this.lastName = lastName.substring(0, 1).toUpperCase() + lastName.substring(1);
```

Comment:
It's not a good idea to hide such login in the constructor. Further, it is a data class, it should not be responsible for capitalizing strings

Reply:
I'll fix this